packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -205,11 +205,32 @@ export class HorizontalScroll extends FASTElement {
         });
     };
 
+    /**
+     * Looks for slots and uses child nodes instead
+     * @internal
+     */
+    private updateScrollStops(): void {
+        const updatedItems: HTMLElement[] = [];
+
+        this.scrollItems.map(item => {
```

Comment:
Minor nit, I'd recommend using `forEach` instead of `map` here because you're not actually returning a value, which is the defining feature between `map` and `forEach`